### PR TITLE
Add policy to ECS task role

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: modules/consignment-api/ecs.tf [hunk 1]
```diff
@@ -1,6 +1,9 @@
 locals {
   app_port = 8080
 }
+
+data "aws_caller_identity" "current" {}
```
Comment (TomJKing) [inline modules/consignment-api/ecs.tf:5]: Is it worth putting this in a `data.tf`, to keep things consistent?

File: modules/consignment-api/ecs.tf [hunk 2]
```diff
@@ -78,6 +80,16 @@ resource "aws_iam_role" "consignment_api_ecs_execution" {
   )
 }
 
+resource "aws_iam_role_policy_attachment" "assume_iam_auth_attach" {
```
Comment (TomJKing) [inline modules/consignment-api/ecs.tf:83]: Don't think the `_attach` is needed on the end of the name, for consistency

File: modules/consignment-api/templates/ecs_assume_role_policy.json.tpl [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ecs-tasks.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
```
Comment (TomJKing) [inline modules/consignment-api/templates/ecs_assume_role_policy.json.tpl:10]: Is the `Sid` field needed, as there is no value for it?

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor comments

File: modules/consignment-api/ecs.tf (hunk 1 above)
Comment (MancunianSam) [reply modules/consignment-api/ecs.tf:5]: Yeah, should do, I was being lazy.

File: modules/consignment-api/ecs.tf (hunk 2 above)
Comment (MancunianSam) [reply modules/consignment-api/ecs.tf:83]: I've taken it off.

File: modules/consignment-api/templates/ecs_assume_role_policy.json.tpl (hunk 3 above)
Comment (MancunianSam) [reply modules/consignment-api/templates/ecs_assume_role_policy.json.tpl:10]: No you're right, bad copy and paste job.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**